README.md

```diff
@@ -58,6 +58,14 @@ An RFC is ready to be approved and finalized once it's Pull Request is ready for
 
 At this time, some member of the core team will motion for a final comment period (FCP). This follows our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process. Once the final comment period has elapsed the RFC will be merged if there are no objections.
 
+## RFC Meetings
+
+RFCs advance through the stages during RFC meetings with TSC and maintainers. Voting follows the [RFC proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
+
+Meetings occur ad hoc rather than on a scheduled basis. They are called for when a proposal author or champion feels it is ready to advance to the next stage. The author or champion can ask for a meeting by contacting TSC to schedule a time.
+
+All maintainers are invited to the meeting. If consensus is reached the RFC advances to the next stage. See [RFC proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) documentation for full details.
```

Comment:
```suggestion
All maintainers are invited to the meeting. If consensus is reached, the RFC advances to the next stage. See [RFC proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) documentation for full details.
```
Should we also say what happens if consensus is *not* reached? Is there a waiting limit before asking again? Is the proposal officially dead?

Reply:
The governance document talks about that some: https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals

Was trying not to repeat too much here that's already covered in the governance doc.